scripts/test_updates.sh

```diff
@@ -24,33 +27,42 @@ exit 1
 }
 
 docker rm -f timescaledb-orig timescaledb-updated timescaledb-clean || true
-rm -rf  ${PGTEST_TMPDIR}/pg_data ${PGTEST_TMPDIR}/pg_data_clean
+rm -rf  ${CLEAN_DATA_DIR} ${UPDATED_DATA_DIR}
 IMAGE_NAME=update_test TAG_NAME=latest bash scripts/docker-build.sh
 
-docker run -d --name timescaledb-clean -v /tmp/pg_data_clean:/var/lib/postgresql/data -p ${CLEAN_PG_PORT}:5432 update_test:latest
-docker run -d --name timescaledb-orig -v ${PGTEST_TMPDIR}/pg_data:/var/lib/postgresql/data -p ${UPDATE_PG_PORT}:5432 timescale/timescaledb:${UPDATE_FROM_TAG}
+docker run -d --name timescaledb-orig -v ${UPDATED_DATA_DIR}:/var/lib/postgresql/data -p ${UPDATE_PG_PORT}:5432 timescale/timescaledb:${UPDATE_FROM_TAG}
+docker run -d --name timescaledb-clean -v ${CLEAN_DATA_DIR}:/var/lib/postgresql/data -p ${CLEAN_PG_PORT}:5432 update_test:latest
```

Comment:
If we are anyhow using Docker, why depend on local filesystem directories (instead of Docker volumes) and local `psql`? I'd just mount the local files/directories needed in the container and instead run everything in the container to ensure consistency and compatibility with more OS:es (e.g., Windows). Seems like a slightly weird mix of Docker and local stuff now.